integration_tests/integration/part-b.spec.js

```diff
@@ -34,6 +34,8 @@ context('Part B', () => {
     cy.uploadEmail({ field: 'emailFileName' })
     cy.clickButton('Continue')
 
+    cy.task('expectUnassignAssessment')
```

Comment:
Can we rename this as `expectUnassignAssessment`?

Reply:
not sure I can see the difference 👀